### PR TITLE
[Bitsail][Docs]Fix Slack URL & Add Official website to README.md.

### PR DESCRIPTION
**Pull request by love-star**
Signed-off-by:

## Pre-Checklist

Note: Please complete **_ALL_** items in the following checklist.

- [v] I have read through the [CONTRIBUTING.md](https://github.com/bytedance/bitsail/blob/master/docs/contributing.md) documentation.
- [v] My code has the necessary comments and documentation (if needed).
- [v] I have added relevant tests.

## Purpose

- Fix Slack URL
- Add Official website to README.md.
- Fix the Expired URL(rebase with [QiLuo-BD])

## Approaches

- Fix Slack URL
- Add Official website to README.md.
- Fix the Expired URL(rebase with [QiLuo-BD])

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,11 +25,13 @@ English | [简体中文](README_zh.md)
...
 [![License](https://img.shields.io/badge/license-Apache%202-4EB1BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
-[![Join Slack](https://img.shields.io/badge/slack-%23BitSail-72eff8?logo=slack&color=5DADE2&label=Join%20Slack)](https://join.slack.com/t/slack-ted3816/shared_invite/zt-1inff2sip-u7Ej_o73sUgdpJAvqwlEwQ)
+[![Join Slack](https://img.shields.io/badge/slack-%23BitSail-72eff8?logo=slack&color=5DADE2&label=Join%20Slack)](https://join.slack.com/t/bitsailworkspace/shared_invite/zt-1l1vgcnlj-gPSWqggOeRHrSO5l7na2WQ)
 
 ## Introduction
 BitSail is ByteDance's open source data integration engine which is based on distributed architecture and provides high performance. It supports data synchronization between multiple heterogeneous data sources, and provides global data integration solutions in batch, streaming, and incremental scenarios. At present, it serves almost all business lines in ByteDance, such as Douyin, Toutiao, etc., and synchronizes hundreds of trillions of data every day.
 
+Official website of BitSail:  https://bytedance.github.io/bitsail/
```
[the remark this answers is not in the document]
Comment (love-star) [reply README.md:33]: Yes, and do I need delete this website line after add the shields.io icon?

File: website/zh/documents/connectors/README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ dir:
 - [Kafka连接器](kafka/kafka.md)
 - [RocketMQ连接器](rocketmq/rocketmq.md)
 - [Redis连接器](redis/redis.md)
+- [Redis V1连接器](redis/redis-v1.md)
```
[the remark this answers is not in the document]
Comment (love-star) [reply website/zh/documents/connectors/README.md:19]: ok

File: website/en/documents/introduce.md [hunk 3]
```diff
@@ -211,13 +212,19 @@ In the Runtime layer, it supports multiple execution modes, such as yarn, local,
     <td> </td>
     <td>✅</td>
   </tr>
+  <tr>
```
Comment (garyli1019) [inline website/en/documents/introduce.md:215]: missing larksheet

File: website/zh/documents/introduce.md [hunk 4]
```diff
@@ -195,11 +196,17 @@ BitSail目前已被广泛使用,并支持数百万亿的大流量场景。同时
     <td> </td>
     <td>✅</td>
   </tr>
+  <tr>
```
Comment (garyli1019) [inline website/zh/documents/introduce.md:199]: missing larksheet

**Pull request merged**